spec/v2/index.bs

```diff
@@ -1986,7 +1986,9 @@ HTTP/1.1 200 OK
 content-length: 0
 </pre>
 
-The host system upon not accepting the event SHOULD respond with an [=error response=] (see [[#api-error-responses]]).
+If the host system accepts the event but cannot process it due to unavailability of the PF data for the [=ProductFootprintFragment=] in the request, it SHOULD return an [=error response=] (see [[#api-error-responses]]) with the [=error response code=]: 404 and appropriate [=error message=] indicating the unavailability of the PF data for the requested [=ProductFootprintFragment=] value.
+
+The host system upon <strong>not</strong> accepting the event SHOULD respond with an [=error response=] (see [[#api-error-responses]]).
```

Comment:
The option to not accept an event (for whatever reason) seems sufficient to me. At least, it allows the host to tell the requestor it will not process the event and they should expect it being 'fulfilled'. Although it would potentially allow solutions to have Events 'implemented' (and be conformant) without ever accepting an event.